test/test_common.h

```diff
@@ -286,7 +286,8 @@ typedef struct {
                 }                                                                                 \
             }                                                                                     \
         } else {                                                                                  \
-            int _count;                                                                           \
+            int _count = 0;                                                                       \
+            PMIX_HIDE_UNUSED_PARAMS(_count);                                                      \
```

Comment:
So...oneapi is claiming that a variable _must_ be used in _all_ "if" clauses or else it is "unused"???? That strikes me like a compiler bug - it's an absurd requirement. Or am I misreading this?

Reply:
That seems like a weird one to raise a warning about, but without seeing the actual warning, it's hard to tell why it is upset.

However, it seems like a much better change would be to just remove the declaration on line 289 and move it to line 299.  Why declare the variable in a larger scope than needed, especially if you're doing work to mask its use?

Reply:
i like @bwbarrett 's idea. i change it that way and hopefully intel (actually clang) will stop complaining.

Reply:
i think i'll just remove the variable.  looks like it was probably being used for some kind of debugging.